lib/src/objects/start_options.dart

```diff
@@ -23,6 +24,9 @@ class StartOptions {
   /// The desired camera resolution for the scanner.
   final Size? cameraResolution;
 
+  /// Whether the scanner should work also with inverted images.
```

Comment:
This doc comment can be improved.
```suggestion
  /// Whether the scanner should try to detect color-inverted barcodes in every other frame.
```